debian/kolibri-server.init

```diff
@@ -88,9 +88,10 @@ do_stop_uwsgi()
   #   other if a failure occurred
   start-stop-daemon --stop --quiet --retry=TERM/30/KILL/5 --pidfile $PIDFILE_UWSGI --name uwsgi
   RETVAL="$?"
-  [ "$RETVAL" = 2 ] && return 2
   # Many daemons don't delete their pidfiles when they exit.
   rm -f $PIDFILE_UWSGI
+  rm -f /tmp/kolibri_uwsgi.sock
```

Comment:
I don't understand why these lines have been moved here, removing the uwsgi lock before uwsgi stops. What's the reason?

Reply:
UWSGI is stopped further up, I moved the whole logic into `do_stop_uwsgi` -- before it had to be run twice because it was doing `[ "$RETVAL" = 2 ] && return 2` firstly.

Reply:
gotcha, thanks!